vi/cs-229-probability.md

```diff
@@ -0,0 +1,385 @@
+
+&#10230; Độ lệch chuẩn - Độ lệch chuẩn của một biến ngẫu nhiên, thường được kí hiệu σ, là thước đo mức độ phân tán của hàm phân phối của nó so với các đơn vị của biến ngẫu nhiên thực tế. Nó được xác định như sau:
+
+<br>
+
+**30. Transformation of random variables ― Let the variables X and Y be linked by some function. By noting fX and fY the distribution function of X and Y respectively, we have:**
+
+&#10230; Biến đổi các biến ngẫu nhiên - Đặt các biến X và Y được liên kết với nhau bởi một hàm. Kí hiệu fX và fY lần lượt là các phân phối của X và Y, ta có:
+
+<br>
+
+**31. Leibniz integral rule ― Let g be a function of x and potentially c, and a,b boundaries that may depend on c. We have:**
+
+&#10230; Quy tắc tích phân Leibniz - Gọi g là hàm của x và có khả năng c, và a, b là các ranh giới có thể phụ thuộc vào c. Chúng ta có:
+
+<br>
+
+**32. Probability Distributions**
+
+&#10230; Phân bố xác suất
+
+<br>
+
+**33. Chebyshev's inequality ― Let X be a random variable with expected value μ. For k,σ>0, we have the following inequality:**
+
+&#10230; Bất đẳng thức Chebyshev - Gọi X là biến ngẫu nhiên có giá trị kỳ vọng μ. Với k,σ>0, chúng ta có bất đẳng thức sau:
+
+<br>
+
+**34. Main distributions ― Here are the main distributions to have in mind:**
+
+&#10230; Các phân phối chính - Dưới là các phân phối chính cần ghi nhớ:
+
+<br>
+
+**35. [Type, Distribution]**
+
+&#10230; [Loại, Phân phối]
+
+<br>
+
+**36. Jointly Distributed Random Variables**
+
+&#10230; Phân phối đồng thời biến ngẫu nhiên
+
+<br>
+
+**37. Marginal density and cumulative distribution ― From the joint density probability function fXY , we have**
+
+&#10230; Mật độ biên và phân phối tích lũy - Từ hàm phân phối mật độ đồng thời fXY, ta có
+
+<br>
+
+**38. [Case, Marginal density, Cumulative function]**
+
+&#10230; [Trường hợp, Mật độ biên, Hàm tích lũy]
+
+<br>
+
+**39. Conditional density ― The conditional density of X with respect to Y, often noted fX|Y, is defined as follows:**
+
+&#10230; Mật độ có điều kiện - Mật độ có điều kiện của X với Y, thường được kí hiệu là fX|Y, được định nghĩa như sau:
+
+<br>
+
+**40. Independence ― Two random variables X and Y are said to be independent if we have:**
+
+&#10230; Tính chất độc lập - Hai biến ngẫu nhiên X và Y độc lập nếu ta có:
+
+<br>
+
+**41. Covariance ― We define the covariance of two random variables X and Y, that we note σ2XY or more commonly Cov(X,Y), as follows:**
+
+&#10230; Hiệp phương sai - Chúng ta xác định hiệp phương sai của hai biến ngẫu nhiên X và Y, thường được kí hiệu σ2XY hay Cov(X,Y), như sau:
+
+<br>
+
+**42. Correlation ― By noting σX,σY the standard deviations of X and Y, we define the correlation between the random variables X and Y, noted ρXY, as follows:**
+
+&#10230; Mối tương quan ― Kí hiệu σX,σY là độ lệch chuẩn của X và Y, chúng ta xác định mối tương quan giữa X và Y, kí hiệu ρXY, như sau:
```

Comment:
Correlation ⇒ Hệ số tương quan

Reply:
Thank you for your suggestion, I will update my translation